src/App.js

```diff
@@ -14,11 +14,88 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: [],
+  }
+
+  toAdd = (good) => {
+    if (!this.state.selected.includes(good)) {
+      this.setState(prevState => ({
+        selected: [...prevState.selected, good],
+      }));
+    }
+  }
+
+  toRemove = (good) => {
+    this.setState((prevState) => {
+      const i = prevState.selected.indexOf(good);
+
+      if (i !== -1) {
+        prevState.selected.splice(i, 1);
+      }
+
+      return {
+        selected: prevState.selected,
+      };
+    });
+  }
+
+  clearSelection = () => {
+    this.setState({
+      selected: '',
+    });
+  }
+
+  render() {
+    const selectedArr = this.state.selected;
+
+    return (
+      <div className="App">
+        <h1>
+          Selected good:
+          {' '}
+          {selectedArr.length > 0 && selectedArr.join(', ')}
+        </h1>
+        <button
+          type="button"
+          className="button-clear buttons"
+          onClick={this.clearSelection}
+        >
+          Clear
+        </button>
+        <ul className="list">
+          {goodsFromServer.map(good => (
+            <li
+              key={good}
+              className={selectedArr.includes(good) ? 'selected' : ''}
```

Comment:
It's a good case to use ClassNames library. Or 
```suggestion
              className={selectedArr.includes(good) && 'selected'}
```

Reply:
Консоль скаржиться що клас не заданий якщо юзати &&

Reply:
And that's why we need ClassNames library